src/i18n/translations/en.js

```diff
@@ -95,7 +95,7 @@ const en = {
         my_websites: "My Websites",
         name: 'Name',
         name_cannot_be_empty: 'Name cannot be empty.',
-        name_cannot_contain_double_period: "Name can not be the '..' character.",
+        name_cannot_contain_double_period: "Name can nsot be the '..' character.",
```

Comment:
A type here

Reply:
@vineeth-vk11 I've corrected the typo error.